docker-entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
@mcjaeger Thanks for the review!

Sure, I've just added the file header you've mentioned to the `Dockerfile`, `docker-compose.yml` file and the `docker-entrypoint.sh` startup script. They're all squashed into the same commit.